README.md

```diff
@@ -7,22 +7,27 @@ This repository contains LaTeX plugin to write SCn code.
 To start using this plugin clone current repo in the directory with your project
 
 ```sh
-git clone https://github.com/ostis-ai/scn-latex-plugin
+mkdir -p ~/texmf/tex/latex
+git clone https://github.com/ostis-ai/scn-latex-plugin ~/texmf/tex/latex
+texhash
 ```
 
 You can also use it with Docker.
 
 ```sh
-docker run -v </abspath/to/project/folder>:/workdir -it --rm ostis/ostis-standard <main .tex-file name>
+docker build . -t scn-latex-plugin
+docker run -v </abspath/to/project/folder>:/workdir -it --rm scn-latex-plugin
 ```
```

Comment:
Actually we're going to publish this image using a CI pipeline. In fact, it is already pushed at https://hub.docker.com/r/ostis/scntex-builder. We can republish it as `scn-latex-plugin` and instruct people to download it from DockerHub instead of building the image locally.